plugin/storage/integration/integration_test.go

```diff
@@ -149,14 +149,18 @@ func (s *StorageIntegration) testGetLargeSpan(t *testing.T) {
 func (s *StorageIntegration) testGetOperations(t *testing.T) {
 	defer s.cleanUp(t)
 
-	expected := []string{"example-operation-1", "example-operation-3", "example-operation-4"}
+	expected := []spanstore.Operation{
+		{Name: "example-operation-1"},
+		{Name: "example-operation-3"},
+		{Name: "example-operation-4"}}
```

Comment:
hmm, how would this work when some storage impls return SpanKind while others do not?

Reply:
Updated the test to include spanKind, when spanKind is missing, it will be treated as empty string in the Operation struct.